frontend/app/src/App.tsx

```diff
@@ -1879,6 +1879,7 @@ export class App extends PureComponent<Props, State> {
                   ? "initial"
                   : scriptRunState
               }
+              data-test-connection-state={connectionState}
```

Comment:
nit: existing attributes names use the pattern `data-test<our-name>` without hyphens `-` in between. I think it would be nice to align all.

Reply:
I renamed the prop above to `data-test-script-state` 👍

Reply:
We also have some other props aligned with that, e.g. `data-test-scroll-behavior`

Reply:
thank you 👍